### PR TITLE
Add logic and tests for StartCommand [#175]

### PR DESCRIPTION
**Pull request by zzori-theoriginal**

### CODE REVIEW

File: src/ViewModels/src/DeploymentDialog/DeploymentDialogViewModel.cs [hunk 1]
```diff
@@ -131,6 +132,19 @@ public string AppName
...
 
+        public string StartCommand
+        {
+            get => _startCmmd;
+
+            set
+            {
+                _startCmmd = value;
+                RaisePropertyChangedEvent("StartCommand");
+
```
Comment (ccheetham) [inline src/ViewModels/src/DeploymentDialog/DeploymentDialogViewModel.cs:143]: is this blank line intended?

File: src/WpfViews/src/DeploymentDialogView.xaml.cs [hunk 2]
```diff
@@ -5,7 +5,6 @@
 using Tanzu.Toolkit.ViewModels;
 using Tanzu.Toolkit.WpfViews.Commands;
 using Tanzu.Toolkit.WpfViews.ThemeService;
-
```
Comment (ccheetham) [inline src/WpfViews/src/DeploymentDialogView.xaml.cs:8]: need a blank line

**Review by ccheetham — CHANGES_REQUESTED** [review]
nits

File: src/ViewModels/src/DeploymentDialog/DeploymentDialogViewModel.cs (hunk 1 above)
Comment (zzori-theoriginal) [reply src/ViewModels/src/DeploymentDialog/DeploymentDialogViewModel.cs]: done

File: src/WpfViews/src/DeploymentDialogView.xaml.cs (hunk 2 above)
Comment (zzori-theoriginal) [reply src/WpfViews/src/DeploymentDialogView.xaml.cs:8]: done

File: src/ViewModels/src/DeploymentDialog/DeploymentDialogViewModel.cs [hunk 3]
```diff
@@ -683,6 +697,14 @@ private void SetBuildpacksFromManifest(AppManifest appManifest)
...
         }
+        private void SetStartCommandFromManifest(AppManifest appManifest)
+        {
+            var startCmmd = appManifest.Applications[0].Command;
+            if (!string.IsNullOrWhiteSpace(startCmmd))
+            {
+                StartCommand = startCmmd;
+            }
+        }
     }
```
Comment (atwoosnam) [inline src/ViewModels/src/DeploymentDialog/DeploymentDialogViewModel.cs:708]: I'm thinking we might want to set StartCommand to null if it's either null or whitespace to account for:
1. if a manifest is provided with no start command (clear any previous value to sync with the new manifest)
2. if a manifest is provided with whitespace for a start command (don't keep the whitespace in the new manifest)... (is this even a real scenario though?)

```suggestion
        private void SetStartCommandFromManifest(AppManifest appManifest)
        {
            var startCmmd = appManifest.Applications[0].Command;
           
            StartCommand = string.IsNullOrWhiteSpace(startCmmd) ? null : startCmmd;
        }
    }
```

**Review by atwoosnam — CHANGES_REQUESTED** [review]

**Review by atwoosnam — APPROVED** [review]

**Pull request merged**